src/framework/ui/uiwidget.cpp

```diff
@@ -1702,6 +1768,9 @@ void UIWidget::onChildFocusChange(const UIWidgetPtr& focusedChild, const UIWidge
 void UIWidget::onHoverChange(bool hovered)
 {
     callLuaField("onHoverChange", hovered);
+
+    if (hovered)
+        callLuaField("onHovered");
```

Comment:
Is there any reason for this extra event? Couldn't we just check hovered in the lua side?